src/components/Buttons/ImageToggleButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement, ReactNode } from "react";
```

Comment:
Copied from original CanvasWrapper backdrop button. It's reused by both the Backdrop and Channel toggles, so I made a generic reusable component.

<img width="100" height="71" alt="Image" src="https://github.com/user-attachments/assets/24c0b22f-0fa8-4c5c-9a39-753f263fe0e5" />

<img width="76" height="62" alt="Image" src="https://github.com/user-attachments/assets/a9fdfdc7-a3b5-4acf-83d7-0b6f58388049" />